src/components/viewer.js

```diff
@@ -45,15 +48,13 @@ const renderChildren = (nodes) =>
       );
       /* eslint-enable react/prop-types */
     }
-
-    // Wrap ListItems in Appear
-    if (node.type === 'ListItem') {
+    if (node.props.listType) {
+      Tag = (node.props.listType === 'ordered') ? 'ol' : 'ul';
+      console.log(children);
       return (
-        <Appear>
```

Comment:
per Rob, he made a point that the `Appear` will mess with our rendering of PDFs so I removed at his request.